.idea/codeStyleSettings.xml

```diff
@@ -104,5 +104,6 @@
       </value>
     </option>
     <option name="USE_PER_PROJECT_SETTINGS" value="true" />
+    <option name="PREFERRED_PROJECT_CODE_STYLE" value="apex-style" />
   </component>
 </project>
```

Comment:
This file is created by IntelliJ and is supposed not to have final line terminator.